storage/tree_storage.go

```diff
@@ -34,7 +34,9 @@ type TreeTX interface {
 type NodeReader interface {
 	// GetTreeRevisionAtSize returns the max node version for a tree at a particular size.
 	// It is an error to request tree sizes larger than the currently published tree size.
```

Comment:
No it's still true. There can't be any revisions larger than the tree size so it will still error if you request one.